### PR TITLE
[MM-33703] Update README with screenshots 

### PR DESCRIPTION
**Pull request by jfrerich**
#### Summary
This PR adds screenshots for subscriptions.  

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-33703

### CODE REVIEW

**Review by mickmister — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -72,12 +72,24 @@ Note, the access token is only used to read ticket information when a subcriptio
...
 
 Triggers send a notification based on specified conditions. Subscriptions currently support the `changed` action on a limited number of fields, but will have enhancements in the future.
 
 Zendesk Admins are able to view these subscriptions inside Zendesk via **Settings > Business Rules > Triggers** and all generated Mattermost Zendesk App Trigger names are prefixed with `__mm_webhook__`. After creating a notification from Mattermost, you can access the trigger in Zendesk and modify conditions of the trigger. If an unsupported condition is added, the user will be notified through Mattermost and a link will be provided to the Zendesk trigger where it can be further modified.
 
+After saving a subscription an ephemeral confirmation message is posted with a link to the trigger inside Zendesk.
+
+![subscription post](./docs/subscription-post.png)
+
+When a subscription for status changes is subscribed to a Mattermost channel the following message will be posted that shows in formation about the new ticket.
```
[the remark this answers is not in the document]
Comment (jfrerich) [reply README.md]: done. Thanks!

File: README.md [hunk 2]
```diff
@@ -72,12 +72,24 @@ Note, the access token is only used to read ticket information when a subcriptio
...
 
 Triggers send a notification based on specified conditions. Subscriptions currently support the `changed` action on a limited number of fields, but will have enhancements in the future.
 
 Zendesk Admins are able to view these subscriptions inside Zendesk via **Settings > Business Rules > Triggers** and all generated Mattermost Zendesk App Trigger names are prefixed with `__mm_webhook__`. After creating a notification from Mattermost, you can access the trigger in Zendesk and modify conditions of the trigger. If an unsupported condition is added, the user will be notified through Mattermost and a link will be provided to the Zendesk trigger where it can be further modified.
 
+After saving a subscription an ephemeral confirmation message is posted with a link to the trigger inside Zendesk.
```
[the remark this answers is not in the document]
Comment (jfrerich) [reply README.md]: This is better. Thanks!

File: README.md [hunk 3]
```diff
@@ -72,12 +72,24 @@ Note, the access token is only used to read ticket information when a subcriptio
...
+
+![subscription post](./docs/subscription-post.png)
+
+When a subscription for status changes is subscribed to a Mattermost channel the following message will be posted that shows in formation about the new ticket.
+
+![subscription new ticket](./docs/subscription-notification-new-ticket.png)
+
+When a notification for a subscribed field changes is received, the Zendesk app posts a message in the channel. The message contains a link to the ticket, the field that changed and its previous and changed values.
```
[the remark this answers is not in the document]
Comment (jfrerich) [reply README.md]: Also better wording. Thanks.

**Review by justinegeffen — APPROVED** [review]

**Comment by justinegeffen** [comment]
I can't see the images in preview but am merging anyway so I can check in prod (sometimes the images don't preview for some reason).

**Comment by jfrerich** [comment]
Thanks, @justinegeffen!

**Pull request merged**